tests/fixtures/crypto_metadata.yaml

```diff
@@ -0,0 +1,24 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept: ['*/*']
+      Accept-Encoding: ['gzip, deflate']
+      Authorization: [Token 0000000000000000000000000000000000000000]
+      Connection: [keep-alive]
+      Content-Type: [application/json]
+      User-Agent: [tiingo-python-client 0.3.2]
+    method: GET
+    uri: https://api.tiingo.com/tiingo/crypto/?tickers=btcusd,fldcbtc
+  response:
+    body: {string: '[{"name":"Bitcoin Tether (BTC/USD)","description":"Tether (BTC/USD)","quoteCurrency":"usd","ticker":"btcusd","baseCurrency":"btc"},{"name":"FoldingCoin (FLDC/BTC)","description":"FoldingCoin (FLDC/BTC)","quoteCurrency":"btc","ticker":"fldcbtc","baseCurrency":"fldc"}]'}
+    headers:
+      Allow: ['GET, HEAD, OPTIONS']
+      Content-Length: ['62']
+      Content-Type: [application/json]
+      Date: ['Sun, 08 Oct 2017 10:21:32 GMT']
+      Server: [nginx/1.10.1]
+      Vary: [Cookie]
+      X-Frame-Options: [SAMEORIGIN]
+    status: {code: 403, message: Forbidden}
```

Comment:
Hm, I know these are fixtures, but it's a little surprising that the returned status is `403` and yet we still got this data back. We might want these to be `{code: 200, message: OK}` since we're writing for the "best case" scenario. Same goes for the other two fixtures.